CoreBots.cs

```diff
@@ -1743,7 +1743,7 @@ public void JumpWait()
     /// <param name="ignoreCheck">If set to true, the bot will not check if the player is already in the given room</param>
     public void Join(string map, string cell = "Enter", string pad = "Spawn", bool publicRoom = false, bool ignoreCheck = false)
     {
-        map = map.ToLower() == "tercess" ? "tercessuinotlim" : map.ToLower();
+        map = map.ToLower() == "tercess" ? "tercessuinotlim" : map.ToLower(); map = map.Replace(" ", "");
```

Comment:
what.. were u tryna do here?

Reply:
this will reduce typo stuff for example if some one typed join("AshfallCamp ") or ("Ashfall Camp ") will eventualy send ("ashfallcamp")